test/commands_map.ts

```diff
@@ -406,25 +406,36 @@ var commands_map: CommandValuesMap = {
         client: function () {
             return (((document.querySelector('.wc-shellinput') as HTMLInputElement).placeholder === 'Type your message...'));
         }
+    },
+    "focus on type": {
```

Comment:
Very cool :)
Can we also add a test for non-focus on type, when typing in an AdaptiveCard textbox?

Reply:
Done.